src/utilities/validation.ts

```diff
@@ -9,9 +9,9 @@ export enum Validation {
 }
 
 export const validationClassMap: Record<Validation, string> = {
-    [Validation.Default]: 'tw-border-line-strong',
+    [Validation.Default]: 'tw-border-line-strong tw-text-black-80',
```

Comment:
We should have alias-tokens for that, no? if not, ask Design about that, please.